### ISSUE TITLE
Generate a file where all documented symbols are listed (documented.json)

### ISSUE THREAD

**Issue by erdnussflips** [realm/jazzy#1085]
It would be useful to generate a "documented.json" file parallel to the "undocumented.json" file, in which all symbols that are already documented are listed.

**Comment by rashenoy** [comment]
Is there any way to do it currently, using some other external tool ?
Trying to include live search as mentioned here.. https://pspdfkit.com/blog/2016/adding-live-search-to-jazzy/
However, it needs a documented .json in order to enable it.

**Comment by johnfairh** [comment]
Yes you could generate a documented.json file by processing the 'search.json'.

What do you mean _it doesnt really do the search_ ?  Search is in the `apple` and `fullwidth` theme.

<img width="1054" alt="Screenshot 2021-03-01 at 10 48 57" src="https://user-images.githubusercontent.com/26768470/109487194-ccc59900-7a7b-11eb-90a4-998d852ce305.png">

**Comment by rashenoy** [comment]
Thanks for the update. I am using apple theme. When I type something in the search window it doesnt bring down the list like you have shown. If I hit enter, it just opens the search.json.

**Comment by johnfairh** [comment]
that means you are using file:/// urls and have not disabled local file restrictions

**Comment by rashenoy** [comment]
Thanks. Since I push the documentation (html tree) to our server, what changes are needed during Jazzy build ?

**Comment by johnfairh** [comment]
no changes are needed - have you tried the search on a server yet?

**Comment by rashenoy** [comment]
yes. Nothing happens when I enter the text in the search box.

**Comment by johnfairh** [comment]
if you're sure you've uploaded all the files and are definitely using an http url to access it then you should open an issue with a demo -- given that the search works widely though, it is likely to be a configuration/browser issue

**Comment by rashenoy** [comment]
the documentation is on a server under https. Is that a problem ? If so what changes are required on the server side ?

**Comment by johnfairh** [comment]
no, https is fine

**Comment by rashenoy** [comment]
Thanks for the support. will check internally if there are any restrictions on js.
